src/com/facebook/buck/android/AndroidBinaryGraphEnhancer.java

```diff
@@ -597,7 +600,8 @@ PreDexMerge createPreDexMergeRule(
         allPreDexDeps,
         dexForUberRDotJava,
         dxExecutorService,
-        xzCompressionLevel);
+        xzCompressionLevel,
+        dxConfig.getDxMaxHeapSize());
```

Comment:
@dreiss Do you want want me to change this or is there any other change you requested?